modules/opensimplex/open_simplex_noise.cpp

```diff
@@ -173,6 +173,7 @@ void OpenSimplexNoise::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_image", "width", "height"), &OpenSimplexNoise::get_image);
 	ClassDB::bind_method(D_METHOD("get_seamless_image", "size"), &OpenSimplexNoise::get_seamless_image);
 
+    ClassDB::bind_method(D_METHOD("get_noise_1d", "x"), &OpenSimplexNoise::get_noise_1d);
```

Comment:
Looks like misformed string, use clang-format to avoid such errors

Reply:
Oops, fixed it.